Tasks/Apiatsionak/Task 2/List.py

```diff
@@ -0,0 +1,24 @@
+i = True
+while i:
+    try:
+        buch = {9103976271:[("Reina", "Meinhard"), ("Memphis", "Tennessee")],
+        4199392609:[("Stephanie", "Bruce"), ("Greensboro", "North Carolina")],
+        9099459979:[("Ermes", "Angela"), ("Dallas", "Texas")],
+        6123479367:[("Lorenza", "Takuya"), ("Indianapolis", "Indiana")],
+        7548993768:[("Margarete", "Quintin"), ("Raleigh", "North Carolina")]}
+        a = (input('Enter the number to search for data \n'))
+        c = str(a).replace(' ', '')
+        z = len(str(c))
+        q = str(a).replace(' ', '').isdigit()
```

Comment:
most of these variables are redundant, because the values are being used only once